Fabric.Authorization.API/scripts/AzureAD-Groups-Migration.ps1

```diff
@@ -217,7 +217,6 @@ function Move-ActiveDirectoryGroupsToAzureAD {
                     Write-DosMessage -Level Information -Message "Found group $($azureADGroup.DisplayName) with ObjectId $($azureADGroup.ObjectId) in Azure AD Tenant $($tenantId)"
                 }
                 else {
```

Comment:
this else can probably removed right?

Reply:
It prevents the second `if` check below. So I suppose we could remove that check or this `else`.

Reply:
Ah, I assumed the catch was the end of the foreach